src/Typed/Resources/assets/src/controller.js

```diff
@@ -0,0 +1,128 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+'use strict';
+
+import { Controller } from 'stimulus';
+import Typed from "typed.js";
+
+export default class extends Controller {
+
+    static values = {
+        strings: Array,
+        typeSpeed: Number,
+        smartBackspace: Boolean,
+        startDelay: Number,
+        backSpeed: Number,
+        shuffle: Boolean,
+        backDelay: Number,
+        fadeOut: Boolean,
+        fadeOutClass: String,
+        fadeOutDelay: Number,
+        loop: Boolean,
+        loopCount: Number,
+        showCursor: Boolean,
+        cursorChar: String,
+        autoInsertCss: Boolean,
+        attr: String,
+        bindInputFocusEvents: Boolean,
+        contentType: String,
+    }
+
+    connect() {
+        const options = {
+            strings: this.stringsValue || null,
+            typeSpeed: this.typeSpeedValue || 30,
+            smartBackspace: this.smartBackspaceValue || true,
+            startDelay: this.startDelayValue || 0,
+            backSpeed: this.backSpeedValue || 0,
+            shuffle: this.shuffleValue || false,
+            backDelay: this.backDelayValue || 700,
+            fadeOut: this.fadeOutValue || false,
+            fadeOutClass: this.fadeOutClassValue || 'typed-fade-out',
+            fadeOutDelay: this.fadeOutDelayValue || 500,
+            loop: this.loopValue || false,
+            loopCount: this.loopCountValue || Infinity,
+            showCursor: this.showCursorValue || true,
+            cursorChar: this.cursorCharValue || '|',
+            autoInsertCss: this.autoInsertCssValue || true,
+            attr: this.attrValue || null,
+            bindInputFocusEvents: this.bindInputFocusEventsValue || false,
+            contentType: this.contentTypeValue || 'html',
+            onBegin: this.onBegin,
+            onComplete: this.onComplete,
+            preStringTyped: this.preStringTyped,
+            onStringTyped: this.onStringTyped,
+            onLastStringBackspaced: this.onLastStringBackspaced,
+            onTypingPaused: this.onTypingPaused,
+            onTypingResumed: this.onTypingResumed,
+            onReset: this.onReset,
+            onStop: this.onStop,
+            onStart: this.onStart,
+            onDestroy: this.onDestroy,
+        };
+
+        const typed = new Typed(this.element, options);
+        this._object = typed;
+        this._dispatchEvent('typed:connect', { typed, options });
+    }
+
+    toggle(event)
+    {
+        console.log('toggle')
+        this._object.toggle();
+    }
```

Comment:
What is the purpose of these methods? Are you thinking that someone might, for example, do something like this?

```html
<button data-action="{{ stimulus_action('symfony/ux-typed/typed', 'toggle') }}"
```

If so, that makes sense. But I'm not sure `destroy()` makes sense below, unless there is a use-case you can mention.

Reply:
Yes; that's the idea. But I am not sure this can be done so simply...
I remove those lines for now.